public-api/src/test/java/org/pmiops/workbench/cdr/DataBrowserControllerTest.java

```diff
@@ -287,10 +288,11 @@ public void testGetDbDomains() throws Exception{
     @Test
     public void testConceptSearchWithEmptyQuery() throws Exception{
```

Comment:
We need tests with the various standard concept filters too.

Reply:
Adding some.